firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarState.swift

```diff
@@ -169,6 +170,9 @@ struct ToolbarState: ScreenState, Equatable {
             SearchEngineSelectionMiddlewareActionType.didClearAlternativeSearchEngine:
             return handleSearchEngineSelectionAction(state: state, action: action)
 
+        case HomepageMiddlewareActionType.configuredSearchBar:
+            return handleConfiguredHomepageSearchBarAction(state: state, action: action)
```

Comment:
I needed to add this because the default case does not go to through the reducer